src/main/flight/pid.c

```diff
@@ -849,7 +849,7 @@ static void NOINLINE pidTurnAssistant(pidState_t *pidState)
     targetRates.x = 0.0f;
     targetRates.y = 0.0f;
 
-    if (STATE(AIRPLANE)) {
+    if (STATE(AIRPLANE) && (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE) || navigationRequiresTurnAssistance())) {
```

Comment:
This breaks `TURN_ASSIST` in `ACRO`. Additionally, call to `pidTurnAssist` is guarded by `FLIGHT_MODE(TURN_ASSISTANT) || navigationRequiresTurnAssistance()` anyway.

Reply:
Yes that is on purpose, because in ACRO there is no target bank angle. Besides, I have (accidentally) flow with TURN ASSIST on top of ACRO and it doesn't fly very nicely. It feels like it is constantly trying to increase the bank angle when you are pitching up or down.

Reply:
Makes sense, however the decision of calling or not calling `pidTurnAssistant` should be done outside of this function. Inside this function a check for `(STATE(AIRPLANE))` is to apply different logic for fixed wing vs multirotor.

Reply:
Sure no problem. Is there a more compact way of checking if ACRO mode is active? I imagine this check is needed more than just in this case?